server/services/forms/index.js

```diff
@@ -383,8 +387,9 @@ module.exports = {
                 } else {
                     formDefinition = formDefinitions[context.toUpperCase()][workflow.toUpperCase()][action.toUpperCase()];
                 }
+                let customConfig = formDefinition.customConfig;
 
-                let form = await formDefinition.builder.call(this, { data });
+                let form = await formDefinition.builder.call(this, { data, customConfig });
```

Comment:
You can just expand this out? `{ data, config: formDefinition.config}`
Or maybe even `{ data, formDefinition.config}`